package.json

```diff
@@ -46,7 +46,7 @@
     "webpack-plugin"
   ],
   "peerDependencies": {
-    "stylelint": "^8.0.0",
+    "stylelint": "^8.0.0 || ^9.2.0",
```

Comment:
(I see this was from a previous commit - but still worth mentioning) It's curious that this dependency exists within `peerDependencies` and `dependencies` both. That makes the peerDep moot. @JaKXz  can you weight in as to what the intention should be here: do we require the user install it, or do we maintain the dependency?

Reply:
The intent is that the user installs it themselves, as documented in the README - the peer dependency is to indicate what version of style lint is officially supported but really, any version that does not change the promise API is fine.

Reply:
Cool. So we need to remove that from `dependencies` and move it to `devDependencies` for use in testing.

Reply:
So I need to remove it from peerDependencies and keep it only into devDependencies?

Reply:
my apologies - it was the diff that was throwing me off, and made it look like stylelint was both in peerDeps and deps. we're all good on this comment thread